app/models/identity_tijuana/donation.rb

```diff
@@ -0,0 +1,86 @@
+module IdentityTijuana
+  class Donation < ApplicationRecord
+    include ReadWrite
+    self.table_name = 'donations'
+    belongs_to :user
+    has_many :transactions
+
+    scope :updated_donations, -> (last_updated_at, exclude_from) {
+      where('donations.updated_at > ?', last_updated_at)
+        .and(where('donations.updated_at < ?', exclude_from))
+        .order('donations.updated_at')
+        .limit(Settings.tijuana.pull_batch_amount)
+    }
+
+    scope :updated_donations_all, -> (last_updated_at, exclude_from) {
+      where('donations.updated_at > ?', last_updated_at)
+        .and(where('donations.updated_at < ?', exclude_from))
+    }
+
+    def self.import(donation_id, sync_id)
+      donation = Donation.find(donation_id)
+      donation.import(sync_id)
+    end
+
+    def import(sync_id)
+      member = Member.find_by_external_id(:tijuana, user_id)
+      if member.present?
+        if member.ghosting_started?
+          Rails.logger.warn "Tijuana member (#{id}) is ghosted (#{existing.id}), not updating"
+        else
+          regular_donation_id = nil
+          if frequency != 'one_off'
+            regular_donation_hash = {
+              member_id: member.id,
+              # started_at: nil,
+              # ended_at: nil,
+              frequency: frequency,
+              # medium: nil,
+              source: 'tijuana',
+              # initial_amount: nil,
+              current_amount: amount_in_cents / 100.0,
+              # amount_last_changed_at: nil,
```

Comment:
Ditto about donation_upgrades

Reply:
Good call. The amount_last_changed_at and initial_amount fields are now populated using the information from the donation_upgrades table.